app/src/main/java/net/gsantner/markor/ui/NewFileDialog.java

```diff
@@ -157,7 +157,7 @@ public Dialog onCreateDialog(Bundle savedInstanceState) {
                     final byte[] templateContents = getTemplateContent(templateSpinner, basedir, encryptCheckbox.isChecked());
                     shareUtil.writeFile(f, false, (arg_ok, arg_fos) -> {
                         try {
-                            if (f.exists() && f.length() < 5 && templateContents != null) {
```

Comment:
Why do we have < 5? Because it is ok to append a template to a nearly empty file but not to a file with more than a few characters?

I can revert this, if we want to preserve the < 5 logic

Reply:
It's mere for me and others doing testfiles. Like create file to see how / if stuff works, and then overwrite it again with something else.